apps/meteor/client/views/room/components/body/RoomBody.tsx

```diff
@@ -203,6 +205,8 @@ const RoomBody = (): ReactElement => {
 
 	const retentionPolicy = useRetentionPolicy(room);
 
+	useJumpToMessage(msgParameter);
```

Comment:
What if the message it not loaded and it's not on the view in the first load of the room?